src/coreclr/src/jit/morph.cpp

```diff
@@ -13774,14 +13781,6 @@ GenTree* Compiler::fgMorphSmpOpOptional(GenTreeOp* tree)
                 break;
```

Comment:
This is badly formatted.

I believe it should be written:

```
    if (tree->OperIsCopyBlkOp())
    {
        break;
    }
```

Reply:
yeah, if you have a change after the 10000th line than GitHub shows you all incorrect. We have #13359 but it is not clear how to separate these files without doing morp1.cpp, morph2.cpp.